torch/csrc/autograd/anomaly_mode.cpp

```diff
@@ -0,0 +1,50 @@
+#include "torch/csrc/autograd/anomaly_mode.h"
+#include "torch/csrc/utils/auto_gil.h"
+#include "torch/csrc/utils/python_strings.h"
+
+#include "stdexcept"
+#include "iostream"
+
+namespace torch { namespace autograd {
+
+bool AnomalyMode::_enabled = 0;
+PyObject* AnomalyMode::stacktrace_key = THPUtils_packString("stacktrace_");
+
+void AnomalyMode::store_stack(PyObject* metadata) {
+  AutoGIL gil;
+  auto mod = PyImport_ImportModule("traceback");
```

Comment:
easier to just use THPObjectPtr.

Reply:
Ho I did not knew these existed !
Thanks!